### PR TITLE
[flex]: Update documentation

### PR DESCRIPTION
**Pull request by gavindidrichsen**
Signed-off-by: Gavin Didrichsen <gavin.didrichsen@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,76 @@
...
+
+> pkg_deps=(core/flex)
+
+### Use as Tool
+
+#### Installation
+
+To install this plan, you should run the following commands to first install, and then link the binaries this plan creates.
+
+``hab pkg install core/flex --binlink``
```
Comment (MindNumbing) [inline README.md:35]: Can you show the binary installation / linking as it takes place on the CLI?

File: controls/flex_exists.rb [hunk 2]
```diff
@@ -2,21 +2,22 @@
...
     its('stdout') { should_not be_empty }
+    its('stderr') { should be_empty }
   end
 
-  command_relative_path = input('command_relative_path', value: '/bin/flex')
-  command_full_path = plan_installation_directory.stdout.strip + "#{command_relative_path}"
+  command_relative_path = input('command_relative_path', value: 'bin/flex')
+  command_full_path = File.join(plan_installation_directory.stdout.strip, command_relative_path)
   describe file(command_full_path) do
     it { should exist }
```
Comment (MindNumbing) [inline controls/flex_exists.rb:22]: Can you add a check for flex being an executable?

**Review by MindNumbing — CHANGES_REQUESTED** [review]
A few changes, otherwise great.

File: README.md (hunk 1 above)
Comment (gavindidrichsen) [reply README.md:35]: Done.

File: controls/flex_exists.rb (hunk 2 above)
Comment (gavindidrichsen) [reply controls/flex_exists.rb:22]: Done.  Cheers!

**Review by MindNumbing — APPROVED** [review]
Thanks @gavindidrichsen LGTM! 👍

**Pull request merged**